gaitmap_mad/gaitmap_mad/event_detection/_rampp_event_detection.py

```diff
@@ -38,6 +40,8 @@ class RamppEventDetection(_EventDetectionMixin, BaseEventDetection):
         By default, all events ("ic", "tc", "min_vel") are detected.
         If `min_vel` is not detected, the `min_vel_event_list_` output will not be available.
         If "ic" is not detected, the `pre_ic` will also not be available in the output.
+    input_stride_type
```

Comment:
We need to document what happens in case of IC strides as input. Probably something ot put in the Notes section